homeassistant/components/sensor/android_ip_webcam.py

```diff
@@ -78,8 +78,8 @@ def icon(self):
             rounded_level = round(int(self._state), -1)
             returning_icon = 'mdi:battery'
             if rounded_level < 10:
-                returning_icon = 'mdi:battery-outline'
-            elif self._state == 100:
+                returning_icon = 'mdi:battery-alert'
```

Comment:
Can you rewrite this to use [`icon_for_battery_level`](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/util/icon.py#L5). And then update that method to use `battery-alert`.

Also, that method needs to be moved to `helpers` because it is Home Assistant specific.

I just left a similar comment about moving it to helpers so maybe conflicts incoming https://github.com/home-assistant/home-assistant/pull/9594#discussion_r141249519

Reply:
How is the function specific to home assistant?

Reply:
Icons are using the Material Design Icon set, which is what we use. All other things in the util package would be useful for any Python project.

That being said, Pascal just told me he was confused too. So maybe I'm wrong here

Reply:
Didn't know that other method existed! Thanks. I'll wait for a consensus on method location.

Reply:
Ok, fair enough. At least the function is specific for all projects using material design icons. Moving it to helpers is probably correct.

Reply:
Alright, let's move it. Can you do it @happyleavesaoc ?

Reply:
@balloob Yep I can do it. ETA this weekend.

Reply:
@happyleavesaoc  once you move it, please let me know so then I can edit the PR #9594 as well. Thank you! :+1: